src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,251 @@
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  Checkbox,
+  Field,
+  GU,
+  IconCheck,
+  IconCross,
+  Link,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { validateEmail } from '../../utils/notifications-utils'
+import emailIllustration from '../../assets/emailIllustration.svg'
+
+const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
+  updateMode,
+  compactMode,
+  email,
+  onSubscribe,
+  onSubscribeSuccess,
+  onReSendVerificationEmail,
+  onDeleteEmail,
+  onError,
+}) {
+  const theme = useTheme()
+  const [emailAddress, setEmailAddress] = useState('')
+  const [emailInvalid, setEmailInvalid] = useState(null)
+  const [termsAccepted, setTermsAccepted] = useState(false)
+
+  const handleEmailAddressBlur = useCallback(e => {
+    const email = e.target.value
+    setEmailInvalid(!validateEmail(email))
+  }, [])
+
+  const handleEmailAddressChange = useCallback(e => {
+    const email = e.target.value
+    setEmailAddress(email)
+    if (validateEmail(email)) {
+      // Set only as valid while user typing. Use blur to set invalid
+      setEmailInvalid(false)
+    }
+  }, [])
+
+  const handleSubscribe = useCallback(async () => {
+    const { subscribedEmail, error } = await onSubscribe(emailAddress)
+
+    if (error) {
+      onError()
```

Comment:
Not sure what `error` could be here, but should we pass it to `onError()`?

Reply:
ah i raised a concern about the error screens, i left everything prepared just in case we want to show the exact error that happened but right now we only have this general error screen 
https://www.figma.com/file/IdVA6iMn0cWLV5imylwEUP/%E2%9A%96%EF%B8%8F-Court-Dashboard-Master?node-id=4980%3A7830